src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  let newState = { ...state };
+  const result = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    const currentObject = actions[i];
```

Comment:
```suggestion
    const action = actions[i];
```
use meaningful naming